src/quo2/components/wallet/network_bridge/view.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.wallet.network-bridge.view
+  (:require
+    [clojure.string :as string]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.foundations.resources :as resources]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.wallet.network-bridge.style :as style]))
+
+(defn network-bridge-internal
```

Comment:
If I'm not mistaken we should be naming as `view-internal` and `view`.

Reply:
I think we name the function as the namespace name. If we name it `view`, all of our component functions will be named `view` -- doesn't make sense I think.

I checked the codebase I can see there has been confusion among people, some are naming `view` and others are naming as the namespace. I think we need to clear this up.

Reply:
no strong opinions, I think Icaro is right that it is in the guidelines 🤔 

However, we export these via `quo2.core` and there we name them correctly so it's fine. i.e when we use them in the application code it will be `quo/network-bridge` regardless of the naming of `view` or not on the component

And then for uses of components internal to quo2 it is probably nicer if they have `view` because then for example we can do

```
(:require [quo2.components.buttons.button.view :as button])

...
[button/view {} ""]
```

Reply:
@OmarBasem, @J-Son89, this is cleared-up already, just got lost in time a little bit. The cause I think is because we decided on a guideline and updated the docs, but it hasn't been reinforced in PR reviews.

It is the namespace alias the communicates what the component is. The convention works in quo2 and in all status-im2 namespaces too. It always works for files named `view.cljs` and exporting a central/main `view` var.

Just take a look for instance in the namespace `status-im2.navigation.screens` and var `screens`. There we can see some screens expose the var `view` from their namespaces named `blabla/view.cljs` and the code is always unambiguous.